integration-tests/emulated/tests/bridges/bridge-hub-polkadot/src/tests/snowbridge.rs

```diff
@@ -459,6 +460,184 @@ fn send_token_from_ethereum_to_asset_hub() {
 	});
 }
 
+/// Tests sending ether from Ethereum to Asset Hub and back to Ethereum
+#[test]
+fn send_eth_asset_from_asset_hub_to_ethereum() {
+	let assethub_sovereign = BridgeHubPolkadot::sovereign_account_id_of(
+		BridgeHubPolkadot::sibling_location_of(AssetHubPolkadot::para_id()),
+	);
+
+	BridgeHubPolkadot::fund_accounts(vec![
+		(assethub_sovereign.clone(), INITIAL_FUND),
+		(RelayTreasuryPalletAccount::get(), INITIAL_FUND),
+	]);
+	AssetHubPolkadot::fund_accounts(vec![
+		(AssetHubPolkadotReceiver::get(), INITIAL_FUND),
+		(ethereum_sovereign_account(), INITIAL_FUND),
+	]);
```

Comment:
This may not be necessary.

Reply:
The test failed without `(AssetHubPolkadotReceiver::get(), INITIAL_FUND),` so I pulled it in from the WETH test. The fee assertions of the test are what failed.